tips/tips-en.txt

```diff
@@ -89,3 +87,20 @@ https://www.haiku-os.org/docs/userguide/en/preferences/keymap.html
 GUI
 You can easily move windows between workspaces by dragging them around in the Workspaces applet. Switching through workspaces is even faster with the keycombo CTRL + ALT + cursor keys.
 https://www.haiku-os.org/docs/userguide/en/workspaces.html
+%
+Terminal
+Press CTRL R and start to enter a command and you'll be provided with the first match from the bash history. Keep pressing CTRL R until you find the right command line and press ENTER to execute it.
+https://www.haiku-os.org/docs/userguide/en/applications/terminal.html#shortcuts
+%
+GUI
+Use keyboard's PgUp/PgDown and Home/End to quickly navigate the context menu in a very crowded folder.
```

Comment:
Use "Page↑ / Page↓" instead.
Come to think of it, this only works when doing keyboard navigation. I find it actually a bit difficult to do...
Suggestion: Add a tip instead that reveals that holding SHIFT while scrolling with the mouse wheel does page-wise scrolling in Tracker (window and context menu).